function/templates/ScaledObject.yaml

```diff
@@ -1,8 +1,9 @@
-{{- if eq .Values.trigger.blob.connection "" }}
+{{- $authRequired := include "function.auth-required" . -}}
+{{- if eq $authRequired "1" }}
 apiVersion: keda.k8s.io/v1alpha1
 kind: TriggerAuthentication
```

Comment:
Can we move TriggerAuthentication into a separate file pls ? That's usually how we maintained all other charts. one yaml per resource.